styles/style.css

```diff
@@ -239,13 +241,15 @@ div#content_block { width: 70% }
 /* Pinterest Button */
 .rtsocial-pinterest-button,
 #rtsocial-ord-pin label { background-position: 0 -598px; border: 1px solid #c9c5c5; border-color: #E8E4E4 #C9C5C5 #C9C5C5; border-radius: 4px 4px 4px 4px; display: block; height: 22px; text-indent: -9999px; text-transform: uppercase; width: 47px; margin: 0 auto; }
+.rtsocial-pinterest-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -598px no-repeat !important;}
 
 .rtsocial-pinterest-button:hover,
 #rtsocial-ord-pin label:hover{ background-position: 0 -640px; border: 1px solid #B8B4B4; }
 
 /* Linked In Button */
 .rtsocial-linkedin-button,
 #rtsocial-ord-lin label{ background-position: 0 -682px; border-color: #E2E2E2 #BFBFBF #B9B9B9; border-style: solid; border-width: 1px; border-radius: 2px; display: block; height: 18px; margin: 0 auto; text-indent: -9999px; text-transform: uppercase; width: 59px; }
+.rtsocial-linkedin-button:focus  {background: url('../images/rtsocial-sprite-image.png') 0 -682px no-repeat !important;}
```

Comment:
Use of `!important` is causing the PHPCS test failure.

Reply:
The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files

Reply:
The problem can't be solved without using `!important.` Also we should not use `phpcs` to check `css` files